frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -233,6 +245,9 @@ pub mod pallet {
             BalanceOf<T>,
             T::SmartContract,
         ),
+
+        DelegateRewardAccountSet(T::AccountId, T::SmartContract, T::AccountId),
+        DelegateRewardsAccountUnset(T::AccountId, T::SmartContract),
```

Comment:
It would be good to have docs, same as previous enum values.

Reply:
added it 👍